dev/run-tests.py

```diff
@@ -655,7 +655,7 @@ def main():
             opts.parallelism,
             with_coverage=os.environ.get("PYSPARK_CODECOV", "false") == "true",
         )
-        run_python_packaging_tests()
+        # run_python_packaging_tests()
```

Comment:
Yeah, we should probably deduplicate because it runs for every python module test. another way is just to add an env variable, and enable it in only one split.

Reply:
I'm just checking whether disabling this test works.
will do so if this is the root cause.

Reply:
<img width="1026" alt="image" src="https://github.com/apache/spark/assets/15246973/7f4b4b74-26d8-4bb9-aec9-b65367c3de96">

From this workflow: https://github.com/panbingkun/spark/actions/runs/5653268655/job/15314215187 Look, it seems to be a problem with `run_python_packaging_tests`.

Because all the tests in `pyspark-pandas-connect-part-3` have ended running, `run_python_packaging_tests` caused an error

Reply:
I am not 100% sure about this, since in https://github.com/zhengruifeng/spark/actions/runs/5652363908/job/15311889590 I also disabled all the packaging tests. (you can check that there is no packaging tests in `pyspark-core`),
but `pyspark-sql` and `pyspark-pandas-slow-connect` still failed ...


I just rebase the PR to re-test whether disabling works

Reply:
From the test results, after disable `run_python_packaging_tests` and splitting, it was successful.
https://github.com/panbingkun/spark/actions/runs/5654907063/job/15318981268